src/color/shade.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+import mix from './mix'
+
+/**
+ * Shades a color by mixing it with black. Leverages the mix color module.
```

Comment:
@bhough can you remove `Leverages the mix color module`? I think it doesn't add much value for a user, but I really liked your explanation in the comment. What if we do something like this:

```
Shades a color by mixing it with black. Compared to `darken` it can produce hue shifts, wheres `darken` manipulates the luminance channel and therefor doesn't produce hue shifts.
```